addons/account/models/company.py

```diff
@@ -20,6 +20,8 @@ class ResCompany(models.Model):
     bank_account_code_prefix = fields.Char(string='Prefix of the bank accounts', oldname="bank_account_code_char")
     cash_account_code_prefix = fields.Char(string='Prefix of the cash accounts')
     accounts_code_digits = fields.Integer(string='Number of digits in an account code')
+    accounts_sale_tax_id = fields.Many2one('account.tax', string="Default Sale Tax")
```

Comment:
Why `account_sale_tax_id` and not `account_sale_tax_id` or even better `default_sale_tax`?